source/op/pt/CMakeLists.txt

```diff
@@ -20,6 +20,11 @@ endif()
 
 find_package(MPI)
 if(MPI_FOUND)
+  include(CheckCXXSymbolExists)
+  check_cxx_symbol_exists(MPIX_Query_cuda_support "mpi.h" CUDA_AWARE)
```

Comment:
I am wondering whether the MPI directory should be passed (does cmake know it). Do you have a test?